<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommuniWidget</class>
 <widget class="QWidget" name="CommuniWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommuniWidget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>14</y>
     <width>921</width>
     <height>593</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>10</y>
       <width>581</width>
       <height>505</height>
      </rect>
     </property>
     <property name="title">
      <string>Network List</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableWidget_tcp"/>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_send_tcp">
     <property name="geometry">
      <rect>
       <x>602</x>
       <y>18</y>
       <width>203</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>824</x>
       <y>18</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>528</y>
       <width>833</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save_tcp">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Confirm </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel_tcp">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>62</y>
       <width>297</width>
       <height>451</height>
      </rect>
     </property>
     <property name="title">
      <string>Receive</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QTextEdit" name="textEdit_recv_tcp"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Serial port</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_send_com">
     <property name="geometry">
      <rect>
       <x>648</x>
       <y>22</y>
       <width>177</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_send_com">
     <property name="geometry">
      <rect>
       <x>842</x>
       <y>20</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>839</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save_com">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Confirm </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel_com">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>511</height>
      </rect>
     </property>
     <property name="title">
      <string>Serial port list</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableWidget_com"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>66</y>
       <width>273</width>
       <height>455</height>
      </rect>
     </property>
     <property name="title">
      <string>Receive</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QTextEdit" name="textEdit_recv_com"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_save_com</sender>
   <signal>clicked()</signal>
   <receiver>CommuniWidget</receiver>
   <slot>onComBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>CommuniWidget</receiver>
   <slot>onTcpBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel_com</sender>
   <signal>clicked()</signal>
   <receiver>CommuniWidget</receiver>
   <slot>onComBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save_tcp</sender>
   <signal>clicked()</signal>
   <receiver>CommuniWidget</receiver>
   <slot>onTcpBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel_tcp</sender>
   <signal>clicked()</signal>
   <receiver>CommuniWidget</receiver>
   <slot>onTcpBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_send_com</sender>
   <signal>clicked()</signal>
   <receiver>CommuniWidget</receiver>
   <slot>onComBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTcpBtnClicked()</slot>
  <slot>onComBtnClicked()</slot>
 </slots>
</ui>
